lib/pilot/options.rb

```diff
@@ -75,8 +75,16 @@ def self.available_options
                                      is_string: false,
                                      verify_block: proc do |value|
                                        raise "Please enter a valid positive number of seconds" unless value.to_i > 0
-                                     end)
-
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :build_full_version,
+                                     short_option: "-b",
+                                     env_name: "PILOT_BUILD_FULL_VERSION",
+                                     description: "Build version with format version(build_number), ex 1.0(12)",
+                                     is_string: true,
+                                     optional: true,
+                                     verify_block: proc do |value|
+x                                       raise "Please use valid format, ex 1.0(12)" unless (value =~ /.*\(.*\)/)
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not -39) spaces for indentation.<br>Don't use parentheses around the condition of an `unless`.<br>Put one space between the method name and the first argument.